Plugin~/WebRTCPlugin/UnityLogStream.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this comment needed?
The source code license is under UCL.
https://github.com/Unity-Technologies/com.unity.webrtc/blob/develop/LICENSE.md

Reply:
https://github.com/Unity-Technologies/com.unity.webrtc/pull/519/files#diff-1e4a8f07211ad301d32a15a93d1511e033b7d5d56a6586cf3dc393fc9a0fce7bR1

Reply:
removed license header generated by VS